.github/workflows/avm.template.module.yml

```diff
@@ -178,7 +178,10 @@ jobs:
           removeDeployment: "${{ fromJson(inputs.workflowInput).removeDeployment }}"
           customLocation: "${{ fromJson(inputs.workflowInput).customLocation }}"
         env:
+          AZURE_CLIENT_ID: ${{ secrets.AZURE_CLIENT_ID }}
```

Comment:
Unrelated changes. Please undo for PR

Reply:
Done. Please consider reopening/merging https://github.com/Azure/bicep-registry-modules/pull/2523. It contains just the changes mentioned here and does not at all conflict with current workflows.